server/src/main/kotlin/suwayomi/tachidesk/manga/impl/util/DirName.kt

```diff
@@ -44,8 +44,12 @@ private fun getChapterDir(mangaId: Int, chapterId: Int): String {
     return getMangaDir(mangaId) + "/$chapterDir"
 }
 
+fun getThumbnailDownloadPath(): String {
+    return applicationDirs.downloadsRoot + "/thumbnails"
+}
+
 fun getChapterDownloadPath(mangaId: Int, chapterId: Int): String {
-    return applicationDirs.mangaDownloadsRoot + "/" + getChapterDir(mangaId, chapterId)
+    return applicationDirs.downloadsRoot + "/mangas/" + getChapterDir(mangaId, chapterId)
```

Comment:
bad abstraction here

Reply:
added `mangaDownloadsRoot` to app dirs

Reply:
also we should use a sensible path joiner function instead of hand concatenating them